src/main/java/org/cactoos/map/Synced.java

```diff
@@ -150,12 +150,14 @@ public Synced(final Map<X, Y> map,
      */
     public Synced(final Map<X, Y> map) {
         super(
-            new org.cactoos.scalar.Synced<>(
-                () -> {
-                    final Map<X, Y> temp = new ConcurrentHashMap<>(0);
-                    temp.putAll(map);
-                    return temp;
-                }
+            new MapOf<>(
+                new org.cactoos.scalar.Synced<>(
```

Comment:
@andreoss there is no need for wrapping that into Synced anymore since it is evaluated in the constructor

Reply:
@victornoel Then, there's no point in `map.Synced`. Should it be removed?

Reply:
@victornoel Replaced with `Collections.synchronizedMap`